data/pigui/modules/time-window.lua

```diff
@@ -42,10 +42,16 @@ local function displayTimeWindow()
 		if requested == name and current ~= name then
 			color = colors.white
 		end
-		local time = name
 		-- translate only paused, the rest can stay
-		if time == "paused" then
+		local time = name
+		if (name == "paused") then
 			time = lc.PAUSED
+			if ui.isMouseDown(0) and (200 >= (Engine.ticks - lastPause)) then
+				ui.optionsWindow:open()
+			end
+			if ui.isMouseReleased(0) then
+				lastPause = Engine.ticks
+			end
```

Comment:
Please prefer the use of `ui.isMouseDoubleClicked(0)` (refer to pigui/libs/forwarded.lua for precise naming) instead of manually tracking double-click time.

Reply:
done

(it didnt work the first time I tried it, probably because I used (1) as params)